can/interfaces/iscan.py

```diff
@@ -148,11 +152,11 @@ class IscanError(CanError):
 
     def __init__(self, function, error_code, arguments):
         super(IscanError, self).__init__()
-        #: Status code
```

Comment:
Why was this changed? See [Sphinx documentation](http://www.sphinx-doc.org/en/master/ext/autodoc.html#directive-autofunction).

Reply:
Sorry, I misunderstood that. I will revert that.